src/test/java/model/WalkerTest.java

```diff
@@ -54,10 +54,10 @@ public void testSwitchingDirectionTrue() {
 	@Test
 	public void testMove() throws Exception {
 		walker.move();
-		assertEquals(Settings.MONSTER_SPEED , walker.getX(), 0);
+		assertEquals(37.0 , walker.getX(), 0);
```

Comment:
Why would the x coordinate be different now? Jumping is only vertical and looking at the code, it does not influence speed in the x direction. This probably means you broke something, adjusting the test is not always the right solution if it fails.